activities/Write.activity/js/activity.js

```diff
@@ -355,7 +355,7 @@ define([
         	var footer = "</body></html>";
 			var sourceHTML = header+content+footer;
 			var inputData = 'data:application/vnd.ms-word;charset=utf-8;base64,' + btoa(unescape(encodeURIComponent( sourceHTML )));
-			var mimetype = 'application/vnd.ms-word;';
+			var mimetype = 'application/ms-word;';
```

Comment:
Still not good :laughing: it's `application/msword`

Reply:
Sorry for the typo 😅